src/www/start.action.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Copyright 2018 The Outline Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+npm run action src/www/build_cordova
+scripts/environment_json.sh -p dev > www/environment.json
+cordova prepare browser
+
+webpack serve \
+    --mode=development \
```

Comment:
`webpack serve` respects the `--mode=development` override, where `webpack-dev-server` doesn't appear to. They're effectively the same thing otherwise (I had to install `webpack-dev-server` to use this command)

Reply:
But it seems `webpack-dev-server` is still installed in package.json.

Reply:
`webpack serve` is effectively an alias to `webpack-dev-server` that allows for additional webpack config overrides